packages/web/src/js/Toast.ts

```diff
@@ -0,0 +1,100 @@
+import BaseComponent from './BaseComponent';
+import {
+  ATTRIBUTE_ARIA_EXPANDED,
+  ATTRIBUTE_DATA_TARGET,
+  CLASS_NAME_HIDDEN,
+  CLASS_NAME_OPEN,
+  CLASS_NAME_TRANSITIONING,
+} from './constants';
+import { enableDismissTrigger, enableToggleTrigger, executeAfterTransition, SpiritConfig } from './utils';
+import { EventHandler } from './dom';
+
+const NAME = 'toast';
+const DATA_KEY = `${NAME}`;
+const EVENT_KEY = `.${DATA_KEY}`;
+
+const EVENT_HIDE = `hide${EVENT_KEY}`;
+const EVENT_HIDDEN = `hidden${EVENT_KEY}`;
+const EVENT_SHOW = `show${EVENT_KEY}`;
+const EVENT_SHOWN = `shown${EVENT_KEY}`;
+
+class Toast extends BaseComponent {
+  isShown: boolean;
+  triggers: NodeListOf<Element>;
+
+  static get NAME() {
+    return NAME;
+  }
+
+  constructor(element: SpiritElement, config?: SpiritConfig) {
+    super(element, config);
+
+    this.triggers = this.getTrigger();
```

Comment:
```suggestion
    this.triggers = this.getTriggers();
```

According to a naming it seems like you are assigning one to many but the functions returns many.

Reply:
you are right, it will be better - name changed